plum/function.py

```diff
@@ -88,6 +98,35 @@ def extract_signature(f):
     return signature, return_type
 
 
+def extract_default_args(signature, f):
```

Comment:
Would you mind adding a docstring for this function?

Reply:
done. I also renamed it